bokehjs/src/coffee/models/tools/gestures/wheel_pan_tool.coffee

```diff
@@ -6,18 +6,7 @@ import * as p from "../../../core/properties"
 export class WheelPanToolView extends GestureToolView
 
   _scroll: (e) ->
-    # we need a browser-specific multiplier to have similar experiences
-    if navigator.userAgent.toLowerCase().indexOf("firefox") > -1
-      multiplier = 20
-    else
-      multiplier = 1
-
-    if e.originalEvent?.deltaY?
-      delta = -e.originalEvent.deltaY * multiplier
-    else
-      delta = e.bokeh.delta
```

Comment:
@bryevdv, this is the code I spoke about in the meeting. `e.bokeh.delta` was computed by jquery-mousewheel plugin and is as browser cross-compatible as possible. However, the plugin was circumvented by the earlier branch containing `deltaY`. The reason `multiplier` is necessary is that firefox returns delta in the number of lines, but other web browsers return it in pixels (it's also possible for it to be returned in pages, but no one uses this, as far as I know).

Tools that depend on wheel event have to be tested under all web browsers we support, especially safari and IE, and FF and Chrome on non-Linux platforms.